### ISSUE TITLE
How does flutter implement dynamic form

### ISSUE THREAD

**Issue by smile2014** [flutter/flutter#19978]
How does flutter implement dynamic form ？？？

**Comment by smile2014** [comment]
https://github.com/VictorRancesCode/json_to_form

**Comment by smile2014** [comment]
@zoechi   https://github.com/VictorRancesCode/json_to_form

**Comment by goderbauer** [comment]
What is the issue here? Are you asking for something similar to json_to_form to be added to the framework? Right now the issue doesn't seem actionable.

**Comment by smile2014** [comment]
yes，Similar to angular dynamic form

**Comment by smile2014** [comment]
@goderbauer  I think this is a very important feature. For the development of enterprise management applications, if you do not support dynamic form, the interface must be re-released with an input, which will bring a lot of inconvenience.

**Comment by vajohn** [comment]
@smile2014 so true you find that with products that can change at any time, you can hard code certain things. Out of everything this is the only part flutter has let me down on.

**Comment by smile2014** [comment]
@vajohn  yes,It would be perfect if the official website provided an angular dynamic form.

**Comment by nate-thegrate** [comment]
> How does Flutter implement dynamic form?

> Are you asking for something similar to json_to_form to be added to the framework?

> yes, similar to angular dynamic form

I believe the functionality requested by this issue is already present in the [reactive_forms](https://pub.dev/packages/reactive_forms) package.
